vue/src/components/BaseSideBar.vue

```diff
@@ -36,10 +36,14 @@ onGotResponse(async (res) => {
 				color="white" active-class="text-white bg-white/10" :to="route.path">
 				<v-list-item-title :style="{ fontWeight: 700 }"> {{ (route.meta!.text as string) }}</v-list-item-title>
 			</v-list-item>
+			<v-list-item active-class="text-white bg-white/10" target="_blank" rel="noopener noreferrer"
+				href="https://documentation.omnect.conplement.cloud/" class="text-white hover:bg-white/10">
```

Comment:
ok, right now we don't have a omnect-ui app documentation. fine for me to use he root-dir of the docu...

Reply:
link already exists: https://documentation.omnect.conplement.cloud/omnect-Secure-OS/omnect-ui

please change